lib/import_js/vim_editor.rb

```diff
@@ -113,7 +113,9 @@ def ask_for_selection(word, alternatives)
     # Get the preferred max length of a line
     # @return [Number?]
     def max_line_length
-      get_number('&textwidth')
+      length = get_number('&textwidth')
+      return length unless length == 0
```

Comment:
nbd, but I think this could have been a one-liner: `get_number('&textwidth') || 80`